com.cubrid.common.update.feature/feature.properties

```diff
@@ -1,7 +1,7 @@
 featureName=CUBRID Tools Common Update Features
 providerName=Search Solution Corporation
 description=CUBRID Tools Common Update Feature
-copyright=Copyright (C) 2014 Search Solution Corporation. All rights reserved by Search Solution.
+copyright=Copyright (C) 2016 CUBRID Corporation. All rights reserved by Search Solution.
 licence=BSD Licence\n\
 \n\
 Copyright (C) 2014 Search Solution Corporation. All rights reserved by Search Solution.\n\
```

Comment:
Is there a reason why you didn't change this part?

Reply:
same as above